package-lock.json

```diff
@@ -9021,7 +9098,7 @@
       "dev": true
     },
     "wet-boew": {
-      "version": "github:wet-boew/wet-boew#0295e93e2b8d8e0ed41a62923046d87d4ef56081",
```

Comment:
Does this make sense? Running ``npm install`` seems to have bumped up the wet-boew dependency's commit hash from 4.0.28's final commit to 4.0.29-development's most recent commit.

Reply:
Yes, GCWeb should always use the latest version of wet-boew, but do you think we would need to update the GCWeb package.lock file every time we merge a PR in wet-boew?

Reply:
That might be a better question for @nschonni and @LaurentGoderre.

I think what happened here is that since package.json doesn't currently specify a specific version or commit hash for wet-boew, ``npm install`` always ends up installing the latest commit.

From my understanding, package.json is valued more than package-lock.json. So for example, if someone runs ``npm install`` in a project whose package.json file specifies a newer version of a dependency than package-lock.json, package.json will "win" and package-lock.json will get updated to what got installed.

Reply:
Maybe package.json could be modified to specify the tag of the most recent major release of wet-boew and only be modified on an as-needed basis in-between releases. Like changing it to a specific commit hash from time to time if significant things change in wet-boew that could impact GCWeb.

Reply:
Going to revert this line to what it originally was since it's beyond the scope of this PR.